### PR TITLE
Add service list and zero state views

### PR DESCRIPTION
**Pull request by bryk**
This change is mostly boilerplate for the two views and cleanup of test
code that has been laying around.

No actual business logic in this change.

### CODE REVIEW

File: src/app/frontend/service/service.module.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import stateConfig from './service.state';
+
+
+/**
+ * Angular module for the service view.
+ *
+ * The view shows services running in the cluster and allows to manage them.
+ */
+export default angular.module(
+    'kubernetesDashboard.service',
```
Comment (PeWu) [inline src/app/frontend/service/service.module.js]: Let's call this "serviceList"

Comment (bryk) [inline src/app/frontend/service/service.module.js]: Done.

**Comment by bryk** [comment]
@PeWu Updated. PTAL

**Comment by PeWu** [comment]
LGTM

**Pull request merged**